config/application.rb

```diff
@@ -31,5 +31,21 @@ class Application < Rails::Application
 
     # Do not swallow errors in after_commit/after_rollback callbacks.
     config.active_record.raise_in_transactional_callbacks = true
+
+    # Lets fix the generators. What is and what is not going to be generated by 'rails g'
+    config.generators do |g|
+      g.stylesheets false
+      g.javascripts false
+      g.test_framework :rspec,
+                       fixtures: true,
+                       model_specs: true,
+                       view_specs: false,
+                       helper_specs: true,
+                       routing_specs: false,
```

Comment:
I feel that everything we can test in view specs can be tested in Cucumber. I know that I propageted for them in WSO but I kind of changed my mind on this. Routing specs? Please make a case for it and we include them. 

Overall I feel that our main focus should be unit tests and acceptance tests.